schemas/api_schema.js

```diff
@@ -95,6 +95,7 @@ type Query {
     SessionsByUser(user_id: ID!, start: Int!, count: Int!): [Session!]
     ResessionsByCategory(category: ID!, start: Int!, count: Int!): [Resession!]
     ResessionsByUser(user_id: ID!, start: Int!, count: Int!): [Resession!]
+    SearchSessions(keywords: ID!): [Session!]
```

Comment:
No, ID is not supposed to be used like this. Use `[String!]!` as the type instead

Reply:
oh crap